### PR TITLE
Feature/191 mobile navigation

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: src/demo/pages/BlockchainTx.css [hunk 1]
```diff
@@ -191,4 +191,10 @@
   right: -11px;
   height: 5px;
   background-color: var(--primary);
+}
+
+@media only screen and (min-width: 375px) and (max-width: 767px) {
+  .MuiSvgIcon-root {
```
Comment (theblockstalk) [inline src/demo/pages/BlockchainTx.css:197]: use of pascal case here

File: src/demo/pages/UserHome.tsx [hunk 2]
```diff
@@ -34,10 +35,16 @@ const USerHome: React.FC = () => {
...
   return (
-    <ContainerStyle className="home-container">
-      <div className="user-section display-none">
-        <img src={userLogo} alt="userLogo" className="user-logo" />
-        <span>{username}</span>
+    <div className="home-container">
+      <div className="mobile-container">
+        <div className="user-section">
+          <img src={userLogo} alt="userLogo" className="user-logo" />
+          <span>{username}jack tanner</span>
```
Comment (theblockstalk) [inline src/demo/pages/UserHome.tsx:42]: This should not have "jack tanner" here

File: src/demo/pages/BlockchainTx.css (hunk 1 above)
Comment (sadiabbasi) [reply src/demo/pages/BlockchainTx.css:197]: it's the package mui class so i need to use it as it is

Comment (sadiabbasi) [reply src/demo/pages/BlockchainTx.css:197]: if i added custom class with i need to use !Important

File: src/demo/pages/UserHome.tsx (hunk 2 above)
Comment (sadiabbasi) [reply src/demo/pages/UserHome.tsx:42]: ohhh just for test will remove

File: src/demo/layout/MainLayout.tsx [hunk 3]
```diff
@@ -80,9 +80,28 @@ const MainLayout: React.FC<MainLayoutProps> = ({ children }) => {
           </Menu>
         </Sidebar>
       </div>
+      <nav className="bottom-nav mobile-display">
```
[the remark this answers is not in the document]
Comment (sadiabbasi) [reply src/demo/layout/MainLayout.tsx:83]: as we didn't add the navigation like this so i modify the existsing component

[the remark this answers is not in the document]
Comment (sadiabbasi) [reply src/demo/layout/MainLayout.tsx:83]: and i guess react nativation is for expo and react native apps https://reactnavigation.org/

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**